<ui version="4.0" >
 <class>exec_regul</class>
 <widget class="QDialog" name="exec_regul" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Ejecuci√≥n de regularizaciones de cuentas</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5" >
   <item>
    <widget class="QTableView" name="tabla" />
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Ejercicios:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ejerciciocomboBox" />
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Fecha inicial:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="inicialdateEdit" />
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Fecha final:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="finaldateEdit" />
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" >
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Fecha asiento:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="asientodateEdit" />
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ejecutarpushButton" >
       <property name="text" >
        <string>&amp;Ejecutar</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="salirpushButton" >
       <property name="text" >
        <string>&amp;Salir</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabla</tabstop>
  <tabstop>salirpushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>salirpushButton</sender>
   <signal>clicked()</signal>
   <receiver>exec_regul</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>587</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel" >
     <x>643</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
